src/nagios_sql.py

```diff
@@ -412,10 +462,13 @@ def main():
     user = options.user
     password = options.password
     publisher_name = options.publisher_name
+    server_primary = options.server_primary
 
     func = get_func(test)
     if publisher_name:
         result = func(host, user=user, password=password, publisher_name=publisher_name)
+    elif server_primary:
+        result = func(host, user=user, password=password, server_primary=server_primary)   

```

Comment:
Remove the extra spaces
```suggestion
        result = func(host, user=user, password=password, server_primary=server_primary)
```